templates/index.html

```diff
@@ -6,50 +6,16 @@
 {% endblock head %}
 
 {% block content %}
-  <section class="hero">
-    {% set custom_hero = config.extra.custom_hero %}
-
-    {% if custom_hero %}
-      <div class="pyro-container">
-        <div class="pyro">
-          <div class="before"></div>
-          <div class="after"></div>
-        </div>
-      </div>
-      {% if lang == 'no' %}
-        <div class="banner-container center">
-          <h2 class="banner">Vi vant Årets Studentorganisasjon 2023!!🎉</h2>
-        </div>
-        <br/>
-        <p>
-          15. februar var vi på Studentlivkonferansen på Det Akademiske Kvarter. Her ble årets studentorganisasjon 2023 kåret, og vi gikk av med seieren!!💫
-          Dette var helt utrolig gøy! 
-        </p>
-        <p>
-          Med disse midlene kan vi forbedre utstyret og tjenestene våre for å gi en enda bedre opplevelse for alle organisasjonene vi betjener.
-          Tusen takk til juryen for at de har valgt oss, og tusen takk til 
-          <a href="https://www.instagram.com/sammenbergen/" target="_blank">Sammen Bergen</a>, 
-          <a href="https://www.instagram.com/sammenstudentliv/ target="_blank"">Sammen Studentliv</a> og 
-          <a href="https://www.instagram.com/vtvest/" target="_blank">VtVest</a> for en fantastisk konferanse! 😍🙌🏽
-        </p>
-      {% elif lang == 'en' %}
-        <div class="banner-container center">
-          <h2 class="banner">We won Student Organization of the Year 2023!!🎉</h2>
-        </div>
-        <br/>
-        <p>
-          On 15 February we attended the Student Life Conference at Kvarteret. This year's student organization was named, and we walked away with victory!!💫
-          This was incredibly fun!
-        </p>
-        <p>
-          With these funds, we can improve our equipment and services to provide an even better experience for all the organizations we serve.
-          Many thanks to the jury for choosing us, and many thanks to you
-          <a href="https://www.instagram.com/sammenbergen/" target="_blank">Sammen Bergen</a>, 
-          <a href="https://www.instagram.com/sammenstudentliv/" target="_blank">Sammen Studentliv</a> and 
-          <a href="https://www.instagram.com/vtvest/" target="_blank">VtVest</a> for a fantastic conference! 😍🙌🏽
-        </p>
-      {% endif %}
-    {% else %}
+  {% set news = get_section(path="nyheter/_index.md") %}
+  {% set featured_news = news.pages | sort(attribute="date") | reverse | filter(attribute="extra.is_featured", value=true) %}
+  {% if featured_news and lang == 'no' %}
+    {% set post = featured_news[0] %}
+    <div class="banner-container center">
+      <h2 class="banner">{{ post.title }}</h2>
+    </div>
+    {{ post.content | safe  }}
+  {% else %}
```

Comment:
We can rewrite line 9 the one fetching news to do some sorting on lang and then change if online 11 to also work on english

Reply:
Fixed!